sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1238,24 +1238,24 @@ object Radar {
         }
 
         apiClient.ipGeocode(object: RadarApiClient.RadarIpGeocodeApiCallback {
-            override fun onComplete(status: RadarStatus, res: JSONObject?, country: RadarRegion?) {
-                callback.onComplete(status, country)
+            override fun onComplete(status: RadarStatus, res: JSONObject?, address: RadarAddress?) {
+                callback.onComplete(status, address)
             }
         })
     }
 
     /**
-     * Geocodes the device's current IP address, converting IP address to country.
+     * Geocodes the device's current IP address, converting IP address to RadarAddress.
```

Comment:
@kochis Can you say converting IP address to partial address to match the API docs? Same as above